integrations/acquisition/covidcast/test_is_latest_issue.py

```diff
@@ -0,0 +1,346 @@
+    sql = '''SELECT `issue` FROM `signal_latest` where `time_value` = 20200414 '''
+    self._db._cursor.execute(sql)
+    record = self._db._cursor.fetchall()
+    self.assertEqual(record[0][0], 20200416) #new data added
+
+    updateRow = [
+      CovidcastRow('src', 'sig', 'day', 'state', 20200414, 'pa', 
+                  6.5, 2.2, 11.5, nmv, nmv, nmv, 20200414, 2)]  #should not showup
+    self._db.insert_or_update_bulk(updateRow)
+    self._db.run_dbjobs()
+    
+    #check newer issue in signal_latest
+    sql = '''SELECT `issue` FROM `signal_latest` where `time_value` = 20200414  '''
+    self._db._cursor.execute(sql)
+    record2 = self._db._cursor.fetchall()
+    self.assertEqual(record, record2) #same as previous as is_latest did not change
+
+    #dynamic check
+    sql2 = '''SELECT `issue` FROM `signal_history` where `time_value` '''
+    self._db._cursor.execute(sql2)
+    record3 = self._db._cursor.fetchall()
+    self.assertEqual(3,self.totalRows + 1) #ensure 3 added (1 of which refreshed)
+    self.assertEqual(20200416,max(list(record3))[0]) #max of the outputs is 20200416 , extracting from tuple
+    
+    #check older issue not inside latest, empty field
+    sql = '''SELECT * FROM `signal_latest` where `time_value` = 20200414 and `issue` = 20200415 '''
+    self._db._cursor.execute(sql)
+    emptyRecord = list(self._db._cursor.fetchall())
+    empty = []
+    self.assertEqual(empty, emptyRecord)
+      
+  # make sure rows are added to *_dim only when needed
+    #new src, sig (ensure that it is added into *_dim)
+    #old src, sig (ensure that it is NOT added into *_dim)
+    #new geo (added)
+    #old geo (not added)
+  def test_src_sig(self):
+    #BASE CASES
+    rows = [
+      # CovidcastRow('src', 'sig', 'day', 'county', 20200416, 'wa', #not seen
+      CovidcastRow('src', 'sig', 'day', 'state', 20200414, 'pa', # 
+      2, 2, 2, nmv, nmv, nmv, 20200414, 0),
+      CovidcastRow('src', 'sig', 'day', 'county', 20200414, 'ca', # updating previous entry
+      3, 3, 3, nmv, nmv, nmv, 20200414, 0)
+    ]
+    self._db.insert_or_update_bulk(rows)
+    self._db.run_dbjobs()
+    #setting baseline variables
+    self._db._cursor.execute('''SELECT * FROM `geo_dim`''')
+    record = self._db._cursor.fetchall()
+    self.geoDimRows = len(list(record))
+
+    self._db._cursor.execute('''SELECT * FROM `signal_dim`''')
+    record = self._db._cursor.fetchall()
+    self.sigDimRows = len(list(record))
+
+    self._db._cursor.execute('''SELECT * FROM `signal_latest`''')
+    record = self._db._cursor.fetchall()
+    self.sigLatestRows = len(list(record))
+
+    #test not added first
+    with self.subTest(name='older src and sig not added into sig_dim'):
+      oldSrcSig = [
+        CovidcastRow('src', 'sig', 'day', 'state', 20211111, 'pa', #new src, new sig
+                    99, 99, 99, nmv, nmv, nmv, 20211111, 1),
+        CovidcastRow('src', 'sig', 'day', 'county', 20211111, 'ca', #new src, new sig
```

Comment:
maybe use a `geo_value` that doesnt look like a state abbreviation when you have a `geo_type` of "county" (just for reader clarity, it doesnt actually matter for the test)

Reply:
That's a good point! What are some sample `geo_value` normally used?